core/shared/src/main/scala/caseapp/core/help/HelpFormat.scala

```diff
@@ -12,6 +13,7 @@ import dataclass._
   sortGroups: Option[Seq[String] => Seq[String]] = None,
   sortedGroups: Option[Seq[String]] = None,
   hiddenGroups: Option[Seq[String]] = None,
+  filterArgs: Option[Arg => Boolean] = None,
```

Comment:
Better add this at the very end, not to unnecessarily break bin compat.

Reply:
With a `@since` annotation right before.

Reply:
should be fixed now

Reply:
it seems binary compat still breaks on the CI... 🤔